test/TvDbSharper.Tests/LanguagesClientTest.cs

```diff
@@ -12,6 +13,35 @@
 
     public class LanguagesClientTest
     {
+        [Fact]
+        public Task GetAllAsync_IgnoresLanguagesWithNullValues()
```

Comment:
please let me know if this test ideal, I don't normally write tests in c# so I'm not necessarily privy to what is standard/best practices